log4j-core/src/main/java/org/apache/logging/log4j/core/layout/GelfLayout.java

```diff
@@ -690,22 +726,17 @@ static CharSequence formatTimestamp(final long timeMillis) {
         if (timeMillis < 1000) {
             return "0";
         }
-        final StringBuilder builder = getTimestampStringBuilder();
-        builder.append(timeMillis);
-        builder.insert(builder.length() - 3, '.');
+        final StringBuilder builder = new StringBuilder(20);
+        formatTimestampTo(builder, timeMillis);
         return builder;
     }
 
-    private static final ThreadLocal<StringBuilder> timestampStringBuilder = new ThreadLocal<>();
-
-    private static StringBuilder getTimestampStringBuilder() {
```

Comment:
Also got a kick out of how pointless this buffer is when using the buffer provided by `AbstractStringLayout` already. The joys of new APIs!